### PR TITLE
Spark 3.2.0

### PR DESCRIPTION
**Pull request by gallushi**
### How was this patch tested?

pls merge only after #80

### CODE REVIEW

**Review by guykhazma — COMMENTED** [review]
Please update the build to run on pyspark 3.2 as well in here:
https://github.com/xskipper-io/xskipper/blob/fc994c922b2fe645c36b0274b8aacf7c5787c459/.github/workflows/build.yml#L50

File: src/main/scala/io/xskipper/metadatastore/MetadataStoreManager.scala [hunk 1]
```diff
@@ -28,6 +28,9 @@ trait MetadataStoreManager {
     */
   def getType: MetadataStoreManagerType
 
+
+  protected var objectStatsEnabled: Boolean = true
```
Comment (guykhazma) [inline src/main/scala/io/xskipper/metadatastore/MetadataStoreManager.scala:32]: please add documentation on why this is needed both to the site and the PR and reference the relevant issue

File: src/test/scala/io/xskipper/api/util/APITestUtils.scala [hunk 2]
```diff
@@ -21,12 +21,34 @@ object APITestUtils {
...
   def parseAPITestDescriptors(fileName: String, inputLocationPrefix: String
   = getDefaultInputLocationPrefix()): Seq[APITestDescriptor] = {
-    val input = scala.io.Source.fromFile(fileName).mkString
+    val input = readTextFileAsString(fileName)
     val parser = new JSONParser()
     val jsonObject = parser.parse(input).asInstanceOf[JSONObject]
     parseAPITestDescriptors(jsonObject, inputLocationPrefix)
   }
 
+  def parseAPITestDescriptors(json: JSONObject,
```
Comment (guykhazma) [inline src/test/scala/io/xskipper/api/util/APITestUtils.scala:30]: why this test was moved?

**Review by guykhazma — APPROVED** [review]
Some comments

File: src/test/scala/io/xskipper/api/util/APITestUtils.scala (hunk 2 above)
Comment (gallushi) [reply src/test/scala/io/xskipper/api/util/APITestUtils.scala:30]: @guykhazma i reformatted the file and it changed its order.
(btw it's not a test, it's a util function to parse the test descriptors from JSON)

**Pull request merged**